Tasks/AzureFunctionAppV1/taskparameters.ts

```diff
@@ -28,12 +29,17 @@ export class TaskParametersUtility {
         }  
 
         taskParameters.azureEndpoint = await new AzureRMEndpoint(taskParameters.connectedServiceName).getEndpoint();
-        console.log(tl.loc('GotconnectiondetailsforazureRMWebApp0', taskParameters.WebAppName));   
+        console.log(tl.loc('GotconnectiondetailsforazureRMWebApp0', taskParameters.WebAppName));
 
         var appDetails = await this.getWebAppKind(taskParameters);
         taskParameters.ResourceGroupName = appDetails["resourceGroupName"];
         taskParameters.WebAppKind = appDetails["webAppKind"];
+        taskParameters.sku = appDetails["sku"];
```

Comment:
You need not store sku value in taskparameters. directky check from appdetails and set isComsumption

Reply:
It is done.